monster-cards/src/widgets/cards/createCardsList.ts

```diff
@@ -13,15 +13,16 @@ const createCardList = createWidgetBase
 	.mixin({
 		mixin: {
 			classes: [ 'cardList' ],
-			childNodeRenderers: [
-				function(this: CardList): DNode[] {
-					const { cards } = this.state;
+			getChildrenNodes: function(this: CardList): DNode[] {
+				const { cards } = this.state;
```

Comment:
could default to `[]` as you have done elsewhere for consistency, and then the `if` would not be required.